spring-boot/admin/src/main/resources/proxy-docker-compose-template.yml

```diff
@@ -0,0 +1,14 @@
+services:
+
+  coatrack-proxy:
+    image: coatrack/coatrack-proxy:2.0.0-SNAPSHOT
+    container_name: coatrack-proxy
+    ports:
+      - 8088:8088
```

Comment:
Your suggestion would cause the gateway to discard all incoming requests that do not originate from localhost, which would break the Gateways functionality. Or am I missing something?

Reply:
I checked this again and you are right, we want the port 8080 to be open to all requests and not only localhost.
So this issue is solved